### DISCUSSION TITLE
Building CCDA with encompassing encounter

### DISCUSSION THREAD

**Discussion by AlexPlepler** [synthetichealth/synthea#1461] [General]
I want to create CCDAs with acts (labs, meds etc) related to the encounters, in the same file.
In standard CDA EncompassingEncounter attribute is used for this. 
Is it possible to implement it with Synthea?

**Comment by jawalonoski** [comment]
I'm not a CCDA expert, but Synthea does support CCDA. With the command line, specify `--exporter.ccda.export=true` or modify your `src/main/resources/synthea.properties` file:

```properties
exporter.ccda.export = true
```

With those settings, you'll get one file per patient.

If the XML elements or attributes are not to your liking, and you don't mind some light scripting, you can browse and edit all the CCDA templates. See https://github.com/synthetichealth/synthea/tree/master/src/main/resources/templates/ccda. The top-level template is named `ccda.ftl`, which then ends up including the other templates in various places.
